src/System.Private.ServiceModel/tools/scripts/PingWCFService.ps1

```diff
@@ -7,7 +7,7 @@ For ($i=0; $i -le 60; $i++)
 {
     try
     {
-        $request.DownloadFile($Url, "temp.crl")
+        $request.DownloadString($Url)
```

Comment:
It is my understanding this existed only to ping the Util endpoint only to see if an exception occurred.  The resulting file was not used (and was a constant headache to delete).  This could have been changed to DownloadString long ago.